api/context.go

```diff
@@ -107,6 +108,7 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	c := &Context{}
+	c.App = app.Global()
```

Comment:
So is this the spot where DI would be used to populate the contents of the App container?